mocks.ts

```diff
@@ -67,6 +67,7 @@ export default async function setUpMocks(): Promise<void> {
     })
   await Promise.all([
     assessRisksAndNeedsApiMocks.stubGetRiskSummary('CRN24', riskSummaryFactory.build()),
+    assessRisksAndNeedsApiMocks.stubGetRiskSummary('X320741', riskSummaryFactory.build()),
```

Comment:
This can be done in one line
`['CRN24', 'X320741'].forEach(crn =>
       assessRisksAndNeedsApiMocks.stubGetRiskSummary(crn, riskSummaryFactory.build())
    ),`